doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -419,7 +419,8 @@ and ``coqtop``, unless stated otherwise:
   *string* must be :n:`"@setting_name"`.
   See the :ref:`note above <interleave-command-line>` regarding the order
   of command-line options.
-:-compat *version*: same as ``-compat-from Stdlib Coq<version>``
+:-compat *version*: same as ``-compat-from Stdlib Rocq<version>``
+  (or ``Coq`` when version is ``8*``)
```

Comment:
Then this line in the documentation will need to be updated to be fully accurate.